.github/actions/constellation_create/action.yml

```diff
@@ -214,7 +215,13 @@ runs:
             exit 1
         fi
         echo "$(kubectl get nodes -o json | jq '.items | length')/"${NODES_COUNT}" nodes have joined"
-        kubectl wait --for=condition=ready --all nodes --timeout=10m
+        kubectl wait --for=condition=ready --all nodes --timeout=20m
+        kubectlErrorCode=$?
```

Comment:
This won't work due to the shell flags set in the CI. If the above kubectl fails, the step ends. You could use something like

```bash
if ! kubectl ... ; then
```